controllers/usersignup/usersignup_controller_test.go

```diff
@@ -222,9 +276,7 @@ func TestUserSignupWithAutoApprovalWithoutTargetCluster(t *testing.T) {
 		HasLabelWithValue(toolchainv1alpha1.MasterUserRecordOwnerLabelKey, userSignup.Name).
 		HasOriginalSub(userSignup.Spec.OriginalSub).
 		HasUserAccounts(1).
-		HasUserAccountTierName("base").
-		HasUserAccountNamespaceTemplateRefs("base-dev-123abc1", "base-stage-123abc2").
-		HasUserAccountClusterResourceTemplateRefs("base-clusterresources-654321b").
+		HasTier(*baseNSTemplateTier).
 		Get()
```

Comment:
shouldn't we verify also that the Space was created?

Reply:
Done in 228d8b0e203a596dac7481c16ee7163fbdca9417